File: src/test/resources/output/fibonacci.py
# Código generado desde EBDA

n = 10
a = 0
b = 1
i = 0
temp = 0
corriendo = True
print("Serie Fibonacci de 10 terminos:")
print(a)
print(b)
while corriendo:
    temp = a + b
    print(temp)
    a = b
    b = temp
    i += 1
    corriendo = i < n
print('Resultado final: ' + str(b))

File: src/test/resources/output/fibonacci.c
// Código generado desde EBDA
#include <stdio.h>
#include <string.h>

int main() {
    int n = 10;
    int a = 0;
    int b = 1;
    int i = 0;
    int temp = 0;
    int corriendo = 1;
    printf("%s\n", "Serie Fibonacci de 10 terminos:");
    printf("%d\n", a);
    printf("%d\n", b);
    while (corriendo) {
        temp = a + b;
        printf("%d\n", temp);
        a = b;
        b = temp;
        i++;
        corriendo = i < n;
    }
    return b;
    return 0;
}
